README.md

```diff
@@ -92,12 +95,16 @@ except for `libxmlsec1`, which you can install with Homebrew:
 
 ### Running Tests
 
+Make sure you have [PhantomJS](http://phantomjs.org) installed prior to running tests. Install with the following:
+
+    $ brew install phantomjs
```

Comment:
I would perhaps add something like this after the first sentence: "On OS X, PhantomJS can be installed with Homebrew: `brew install phantomjs`. For other platforms, refer to the [PhantomJS download](http://phantomjs.org/download.html) page."